source/coco_grove.rst

```diff
@@ -3,6 +3,12 @@
 COCO Grove (coming soon!)
 =========================
 
-..
-    <Inroduction to this section - Owner: Ashish>
+.. figure:: /_static/coming-soon.png
```

Comment:
The coming soon image is not visible in the preview and it's appearing as a link instead

Reply:
This is working fine on my PC can you make sure you followed the steps properly.